gson/src/test/java/com/google/gson/functional/ReflectionAccessFilterTest.java

```diff
@@ -58,9 +58,20 @@ public void testBlockInaccessibleJava() {
       );
     }
 
-    // But serialization should succeed for classes with only public fields
-    String json = gson.toJson(new Point(1, 2));
-    assertEquals("{\"x\":1,\"y\":2}", json);
+
+    // But serialization should succeed for classes with only public fields.
+    // Not many JDK classes have mutable public fields, thank goodness, but java.awt.Point does.
+    Class<?> pointClass = null;
+    try {
+      pointClass = Class.forName("java.awt.Point");
```

Comment:
Why are you loading this class with `Class.forName`? Was the class not available for you when you tested this (maybe because the module was missing)?

Reply:
Yeah there was a module problem I was seeing locally that didn't seem to be resolved in the obvious way, doing `require static java.desktop`, so I was impatient and just did this instead. As the comment suggests, it's not too concerning if we don't have this test coverage on Java 17.